### DISCUSSION TITLE
Resume Parsing | DB Schema

### DISCUSSION THREAD

**Discussion by PrakVayuz** [milivayuz/PRAGYAN_AILAB#9] [Ideas]
@technicalbirdVayuz 
**Follow up the new approach listed below-**
1) We need to create a new masterSkills collection to store all job-related skills. Once this collection is established, we will parse resumes and identify the skills they contain by querying the masterSkills schema.
2) Similar Approach will be followed for other parameters namely sectionSchema, EducationSchema, CertificateSchema & ProjectSchema & finally there is a jobSchema that uses subdocuments for skills, education, certificates, and projects.

**Comment by technicalbirdVayuz** [comment]
What about timeline?

**Reply by PrakVayuz** [reply]
Timeline - May 29, 2024

**Reply by technicalbirdVayuz** [reply]
Breakup pls?

**Reply by PrakVayuz** [reply]
Schema Creation for different parameters - 1Hrs
Adding data to different documents- 1Hrs
Algorithm - 2-3Hrs
Testing & Deployment-1Hrs
Buffer-2Hrs

**Comment by technicalbirdVayuz** [comment]
Not all pointers covered @PrakVayuz

**Comment by PrakVayuz** [comment]
**Follow ups** 
**Schema Design:** Develop a schema for the Jobs table that includes fields for the job name and skill set. The job skills will be compared with the skills parsed from resumes.
**CRUD Operations**: Implement Create, Read, Update, and Delete (CRUD) operations to manage job listings and their associated skills.
**Match Score Calculation**: Develop a mechanism to calculate a match score based on the comparison of job skills and resume skills, focusing solely on the skill parameter for now.
**Testing and Deployment**: Ensure thorough testing and deploy the system by the target date of May 29, 2024.

**Reply by PrakVayuz** [reply]
New Target submission date of version 1 will be Jun 3,2024.
